docs/source/tutorials/writing-a-plugin-for-hermes.md

```diff
@@ -164,4 +164,8 @@ You can now write plugins for HERMES.
 To fill the plugin with code, you can check our [hermes-plugin-git](https://github.com/softwarepub/hermes-plugin-git) repository.
 There is the code to check the local git history and extract contributors of the given branch.
 
+Now that you have created your own plugin consider publishing it to the 
+[HERMES Marketplace](https://docs.software-metadata.pub/en/latest/plugins/marketplace.html) so that others can use it as
```

Comment:
A relative link into the markdown should be used here. Sphinx does the conversion to the full URL automatically.